mapfishapp/src/main/webapp/app/addons/notes/js/main.js

```diff
@@ -0,0 +1,175 @@
+/*global
+ Ext, OpenLayers, GEOR
+ */
+Ext.namespace("GEOR.Addons");
+
+GEOR.Addons.Notes = Ext.extend(GEOR.Addons.Base, {
+
+    toggleGroup: "notes",
+
+    layer: null,
+    icon: null,
+
+    /**
+     * Method: init
+     */
+    init: function(record) {
+        this.icon = GEOR.config.PATHNAME + "/ws/addons/" + record.get("name").toLowerCase() + "/" + this.options.icon;
+
+        this.layer = new OpenLayers.Layer.Vector('__georchestra_notes', {
+            displayInLayerSwitcher: false,
+            styleMap: new OpenLayers.StyleMap({
+                "default": new OpenLayers.Style(null, {
+                    rules: [new OpenLayers.Rule({
+                        symbolizer: {
+                            "Point": {
+                                "externalGraphic": this.icon,
+                                "graphicWidth": 18,
+                                "graphicHeight": 18
+                            }
+                        }
+                    })]
+                })
+            })
+        });
+        if (OpenLayers.Util.indexOf(this.map.layers, this.layer) < 0) {
+            this.map.addLayer(this.layer);
+        }
+
+        var action = new GeoExt.Action({
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["GeoExt" is not defined.](https://www.codacy.com/app/georchestra/georchestra/file/2334665166/issues/source?bid=3200181&fileBranchId=3206983#l39)